### ISSUE TITLE
Not work well with parts with no sound between scenes

### ISSUE THREAD

**Issue by Ruke805** [kaegi/alass#34]
Something I noticed is that in certain videos it doesn't work well when in one scene to another, with the screen totally black and no sound it can't sync normally, it usually fixes one part of the video, but not others. Looking through the audio spectrum, I noticed that it's always in this type of scene:

![image](https://user-images.githubusercontent.com/62116034/124336837-19c69280-db76-11eb-8c79-25a2b766f608.png)

Well, That something can be investigated in the future

**Comment by chrysn** [comment]
Does this issue occur even when `--no-split` is given? Or do you need split mode?

**Comment by kaegi** [comment]
I don't see a reason why it wouldn't work in this case (with split or no-split; though no-split is much, much more reliable if there really is no split)... Maybe try to use
```
alass video.mkv _ out.srt
```
If you then start `out.srt` with the video you can see what the algorithm tries to synchronize to. There should be no lines in the 'silent' area.

**Comment by Ruke805** [comment]
> I don't see a reason why it wouldn't work in this case (with split or no-split; though no-split is much, much more reliable if there really is no split)... Maybe try to use
> 
> ```
> alass video.mkv _ out.srt
> ```
> 
> If you then start `out.srt` with the video you can see what the algorithm tries to synchronize to. There should be no lines in the 'silent' area.

Yes, should be done this way, but for some reason when it tries to align in this kind of part, it just "skip", and all the rest of the subtitle be unsynchronized. Sometimes the beginning is synchronized, but in another case all of it keeps unsynchronized.

For example, this subtitle can not be synchronized, I've tried a lot of times, and does not work. Even with --no-split to another flag. You can find this file on rarbg, or if you prefer, I can share it with you, just give me an email.

[Fosse.Verdon.S01E04.1080p.AMZN.WEB-DL.DDP5.1.H.264-MZABI.srt.zip](https://github.com/kaegi/alass/files/6987379/Fosse.Verdon.S01E04.1080p.AMZN.WEB-DL.DDP5.1.H.264-MZABI.srt.zip)
